hello/models.py

```diff
@@ -20,12 +20,7 @@ class Review(models.Model):
     post = models.ForeignKey(Post, on_delete=models.CASCADE, related_name='reviews', blank=True, null=True)
```

Comment:
Currently, the review doesn't know about who posted it. That seems like an oversight. :smile: